lib/m2ts/caption-stream.js

```diff
@@ -20,10 +20,13 @@
 var Stream = require('../utils/stream');
 var cea708Parser = require('../tools/caption-packet-parser');
 
-var CaptionStream = function() {
+var CaptionStream = function(options) {
 
   CaptionStream.prototype.init.call(this);
 
+  // cc708 flag, default to true
+  this.cc708_ = typeof options.cc708 === 'boolean' ? options.cc708 : true;
```

Comment:
Might be worth changing the name to be a bit more descriptive. Maybe `include708Captions` or `parse708Captions`.